flask_admin/contrib/sqla/filters.py

```diff
@@ -118,6 +120,38 @@ def operation(self):
         return lazy_gettext('not in list')
 
 
+# Composite filters
+class FilterAndEqual(FilterEqual):
+    def __init__(self, child_filters, name, delimiter=',', options=None, data_type=None):
```

Comment:
This would be more user-friendly with `data_type='select2-tags'` (to indicate to users that they need to use a comma separated list).